frontend/integration-tests/protractor.conf.ts

```diff
@@ -111,6 +111,7 @@ export const config: Config = {
       'tests/performance.scenario.ts',
       'tests/olm/descriptors.scenario.ts',
       'tests/olm/catalog.scenario.ts',
+      'tests/operator-hub/operator-hub.scenario.ts',
```

Comment:
I'd like to wait until ci/prow/console-e2e tests are enabled for the repo before merging to make sure this passes CI.

Requires https://github.com/openshift/release/pull/2380